code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -304,19 +304,19 @@
 		if(op != "")
 			switch(op)
 				if("+")
-					result += val
+					result = (result + val)
```

Comment:
just comment these, so some enterprising future coder doesn't try to get smart or fresh and "fix" them.